src/api/spec/controllers/webui/user_controller_spec.rb

```diff
@@ -161,6 +161,43 @@
       it { expect(user.email).to eq('new_valid@email.es') }
       it { is_expected.to redirect_to user_show_path(user) }
     end
+
+    context "when in LDAP mode" do
+      before do
+        login user
+      end
+
+      it "users are not allowed to change their accounts" do
+        CONFIG['ldap_mode'] = :on
+        post :save, params: { user: user, realname: 'another real name', email: 'new_valid@email.es' }
+        expect(controller).to set_flash[:error]
+        CONFIG['ldap_mode'] = :off
```

Comment:
Otherwise it would apply for other tests as well. But I'll mock it as you requested.